File: CGStudio/FluidStudio/PhysicsLabels.cs
namespace PG {
	public class CSGBoundarySceneCreateLabels
	{
		public const string CommandNameLabel = "CSGBoundarySceneCreate";
		public const string NewIdLabel = "NewId";
	}

	public class CSGBoundarySceneUpdateLabels
	{
		public const string CommandNameLabel = "CSGBoundarySceneUpdate";
		public const string IdLabel = "Id";
		public const string BoxLabel = "Box";
		public const string NameLabel = "Name";
	}

	public class FluidSceneCreateLabels
	{
		public const string CommandNameLabel = "FluidSceneCreateCommand";
		public const string NewIdLabel = "NewId";
	}

	public class FluidSceneUpdateLabels
	{
		public const string CommandNameLabel = "FluidSceneUpdateCommand";
		public const string IdLabel = "Id";
		public const string ParticleSystemIdLabel = "ParticleSystemId";
		public const string StiffnessLabel = "Stiffness";
		public const string ViscosityLabel = "Viscosity";
		public const string IsBoundary = "IsBoundary";
		public const string NameLabel = "Name";
	}

	public class FluidSimulationLabels
	{
		public const string CommandNameLabel = "FluidSimulationCommand";
		public const string FluidSceneIdsLabel = "FluidSceneIds";
		public const string CSGBoundarySceneIdsLabel = "CSGBoundarySceneIds";
		public const string TimeStepLabel = "TimeStep";
	}

	public class PhysicsSolverCreateLabels
	{
		public const string CommandNameLabel = "PhysicsSolverCreateCommand";
		public const string FluidSceneIdsLabel = "FluidSceneIds";
		public const string CSGBoundarySceneIdsLabel = "CSGBoundarySceneIds";
		public const string TimeStepLabel = "TimeStep";
		public const string NameLabel = "Name";
		public const string NewIdLabel = "NewId";
	}

	public class PhysicsSolverUpdateLabels
	{
		public const string CommandNameLabel = "PhysicsSolverUpdateCommand";
		public const string IdLabel = "Id";
		public const string FluidSceneIdsLabel = "FluidSceneIds";
		public const string CSGBoundarySceneIdsLabel = "CSGBoundarySceneIds";
		public const string TimeStepLabel = "TimeStep";
		public const string NameLabel = "Name";
	}

}

File: Blender/CrystalPython/physics/physics_labels.py
class CSGBoundarySceneCreateCommand:
   CommandNameLabel="CSGBoundarySceneCreate"
   NewIdLabel="NewId"
class CSGBoundarySceneUpdateCommand:
   CommandNameLabel="CSGBoundarySceneUpdate"
   IdLabel="Id"
   BoundingBoxLabel="BoundingBox"
   NameLabel="Name"
class EmitterSceneCreateCommand:
   CommandNameLabel="EmitterSceneCreateCommand"
   NewIdLabel="NewId"
class EmitterSceneUpdateCommand:
   CommandNameLabel="EmitterSceneUpdateCommand"
   IdLabel="Id"
   ParticleSystemIdLabel="ParticleSystemId"
   ParticleRadiusLabel="ParticleRadius"
   StiffnessLabel="Stiffness"
   ViscosityLabel="Viscosity"
   InitialVelocityLabel="InitialVelocity"
   DensityLabel="Density"
   NameLabel="Name"
   StartStepLabel="StartStep"
   EndStepLabel="EndStep"
   IntervalLabel="Interval"
class FluidSceneCreateCommand:
   CommandNameLabel="FluidSceneCreateCommand"
   NewIdLabel="NewId"
class FluidSceneToPSCommand:
   CommandNameLabel="FluidSceneToPSCommand"
   FluidIdLabel="FluidId"
   ParticleSystemIdLabel="ParticleSystemId"
class FluidSceneUpdateCommand:
   CommandNameLabel="FluidSceneUpdateCommand"
   IdLabel="Id"
   ParticleSystemIdLabel="ParticleSystemId"
   ParticleRadiusLabel="ParticleRadius"
   StiffnessLabel="Stiffness"
   ViscosityLabel="Viscosity"
   VorticityLabel="Vorticity"
   DensityLabel="Density"
   IsBoundary="IsBoundary"
   NameLabel="Name"
class FluidSimulationCommand:
   CommandNameLabel="FluidSimulationCommand"
   SolverIdLabel="SolverId"
class MeshBoundarySceneCreateCommand:
   CommandNameLabel="MeshBoundarySceneCreate"
   NewIdLabel="NewId"
class MeshBoundarySceneUpdateCommand:
   CommandNameLabel="MeshBoundarySceneUpdate"
   IdLabel="Id"
   MeshIdLabel="MeshId"
   NameLabel="Name"
class MVPSurfaceConstructionCommand:
   CommandNameLabel="MPVSurfaceConstructionCommand"
   VolumeParticleSystemIdLabel="VolumeParticleSystemId"
   MassParticleSystemIdLabel="MassParticleSystemId"
   TriangleMeshIdLabel="TriangleMeshId"
   ParticleRadiusLabel="ParticleRadius"
   ThresholdLabel="Threshold"
class PhysicsSolverCreateCommand:
   CommandNameLabel="PhysicsSolverCreateCommand"
   NewIdLabel="NewId"
class PhysicsSolverExportCommand:
   CommandNameLabel="PhysicsSolverExportCommand"
   FluidIdsLabel="FluidIds"
   FilePathLabel="FilePath"
   DoExportMicroLabel="DoExportMicro"
class PhysicsSolverUpdateCommand:
   CommandNameLabel="PhysicsSolverUpdateCommand"
   IdLabel="Id"
   FluidSceneIdsLabel="FluidSceneIds"
   EmitterSceneIdsLabel="EmitterSceneIds"
   CSGBoundarySceneIdsLabel="CSGBoundarySceneIds"
   EffectLengthLabel="EffectLength"
   ExternalForceLabel="ExternalForce"
   TimeStepLabel="TimeStep"
   NameLabel="Name"
class SPHSurfaceConstructionCommand:
   CommandNameLabel="SPHSurfaceConstructionCommand"
   ParticleSystemIdLabel="ParticleSystemId"
   TriangleMeshIdLabel="TriangleMeshId"
   ParticleRadiusLabel="ParticleRadius"
   CellLengthLabel="CellLength"
   ThresholdLabel="Threshold"
   IsIsotropicLabel="IsIsotoropic"
